pkg/karmadactl/options/global.go

```diff
@@ -49,6 +49,11 @@ func AddKubeConfigFlags(flags *pflag.FlagSet) {
 	flags.StringVar(DefaultConfigFlags.Context, "karmada-context", *DefaultConfigFlags.Context, "The name of the kubeconfig context to use")
 }
 
+// AddNamespaceFlag add namespace flag to the specified FlagSet.
+func AddNamespaceFlag(flags *pflag.FlagSet) {
+	flags.StringVarP(DefaultConfigFlags.Namespace, "namespace", "n", *DefaultConfigFlags.Namespace, "If present, the namespace scope for this CLI request")
```

Comment:
```suggestion
	flags.StringVarP(DefaultConfigFlags.Namespace, "namespace", "n", *DefaultConfigFlags.Namespace, "If present, the namespace scope for this CLI request.")
```
No big deal, but it feels better with a `.` at the end.